src/main/java/fixie/dictionary_service/entity/PartType.java

```diff
@@ -0,0 +1,49 @@
+package fixie.dictionary_service.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+
+import fixie.dictionary_service.dto.PartTypeDTO;
```

Comment:
where do you use this PartTypeDTO in this file? This import is unnecessary. Check all files in your microservice and delete unused imports (they're marked on gray color).

Reply:
fixed